File: belajar-java/belajar-java-dasar/src/Continue.java
public class Continue {
    public static void main(String[] args) {

        // Perulangan for yang dimulai dari 1 hingga 100
        for (var counter = 1; counter <= 100; counter++) {
            // Jika counter adalah bilangan genap, perulangan dilanjutkan ke iterasi berikutnya tanpa menjalankan kode di bawah ini
            if (counter % 2 == 0) {
                continue; // Melompati iterasi jika kondisi terpenuhi
            }

            // Mencetak nilai counter jika merupakan bilangan ganjil
            System.out.println("Perulangan Ganjil " + counter);
        }

    }
}

File: belajar-php/belajar-php-dasar/continue.php
<?php

$a = 1; 

while ($a <= 100){

    if ($a % 2 == 0){
        echo "Ini adalah bilangan genap: " . $a . PHP_EOL;
    }
    $a = $a + 1;
    continue;
    echo "Ini adalah bilangan ganjil: " . $a . PHP_EOL;
}

